examples/chef/sumologic-otel-collector/resources/default.rb

```diff
@@ -58,7 +58,7 @@
 
 
 def get_install_script_command(resource)
-  command_parts = ["bash", INSTALL_SCRIPT_PATH, "--installation-token #{resource.install_token}"]
+  command_parts = ["SUMOLOGIC_INSTALL_TOKEN=#{resource.install_token}", "bash", INSTALL_SCRIPT_PATH]
```

Comment:
is it working if we pass `ENV` in chef as normal command?

Reply:
That's supposed to work, but I replaced it with a more idiomatic approach from Chef's docs.